lib/repo/queryable.ex

```diff
@@ -74,6 +74,29 @@ defmodule ExAudit.Queryable do
     end
   end
 
+  def latest(module, struct) do
+    import Ecto.Query
+
+    query =
+      from(
+        v in version_schema(),
+        order_by: [desc: :recorded_at],
+        limit: 1
+      )
+
+    query =
+      case struct do
+        %{__struct__: struct, id: id} when nil not in [struct, id] ->
+          from(
+            v in query,
+            where: v.entity_id == ^id,
+            where: v.entity_schema == ^struct
+          )
```

Comment:
do we need a 2nd catch in this case statement in case the first does not match?

Reply:
Yeah, I thought about that... I copied this from the `history` function, which doesn't have a fallback. I suppose that's no excuse for poor coding practices though. :-p Anyway, I looked at it but wasn't sure what I should do in the fallback. Maybe just raise with a more helpful message than "no clause matching ..."?